lib/at_sms_cert/Kconfig

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig AT_SMS_CERT
+	bool "AT SMS commands for certification"
+	depends on SOC_NRF9160_SICA
+	help
+	  Provide an implementation of the following SMS-related AT commands for certification purposes:
+	  AT+CPMS
+	  AT+CSMS
+	  AT+CSCA
+	  AT+CSCS
+	  AT+CMGD
+	  AT+CMSS
+	  AT+CMGW
+
+if AT_SMS_CERT
+
+config AT_SMS_CERT_SYS_INIT
+	bool "Initialize during SYS_INIT"
```

Comment:
curious, what would be the alternative ?

If user disables this then how will it be initialized, and can that be done in a safe manner.

Basically what i'm asking is whether it makes sense that this setting is directly user-configurable.

Reply:
Good point, in general the user either wants to compile this feature or not. However it should be possible to toggle the feature at runtime when it's compiled, we'll need a header for that.

Reply:
> However it should be possible to toggle the feature at runtime when it's compiled, we'll need a header for that.

So until that header is ready, maybe we should remove this setting, or as minimal make it prompt-less.
When the header is reader, then we can allow the user to either initialize this through sysinit (this setting) or runtime.